tests/unit/test_wheel.py

```diff
@@ -463,3 +463,10 @@ def test_PATH_check_case_insensitive_on_windows(self):
             assert retval is None
         else:
             assert retval is not None
+
+    def test_trailing_ossep_removal(self):
+        retval = self._template(
+            paths=[''.join([os.path.join('a', 'b'), os.sep])],
```

Comment:
`["/a/b/"]` and `["/a/b/c"]` should be fine, no need to use os.path here.

Reply:
The person using an operating system where `os.sep == '\\'` begs to differ here :-)

Reply:
Hahaha. Whoops. Sorry. 🙈